tasks/managed/extract-index-image/extract-index-image.yaml

```
@@ -35,13 +38,13 @@ spec:
 
         RESULTS_FILE="$(workspaces.data.path)/$(params.resultsDirPath)/extract-index-image-results.json"
 
-        jsonBuildInfo=$(jq -cr .jsonBuildInfo "$(params.inputDataFile)")
-
-        indexImage=$(echo "$jsonBuildInfo" | jq -cr .index_image)
-        echo -n "$indexImage" > "$(results.indexImage.path)"
+        indexImage="$(jq -r '.components[].index_image' \
+                    "$(workspaces.data.path)/$(params.internalRequestResultsFile)")"
+        echo -en "$indexImage" | tee "$(results.indexImage.path)"
 
-        indexImageResolved=$(echo "$jsonBuildInfo" | jq -cr .index_image_resolved)
-        echo -n "$indexImageResolved" > "$(results.indexImageResolved.path)"
+        indexImageResolved="$(jq -r '.components[].index_image_resolved' \
+                            "$(workspaces.data.path)/$(params.internalRequestResultsFile)")"
+        echo -en "$indexImageResolved" | tee -a "$(results.indexImageResolved.path)"
```

Hmm, I was commenting about this this morning, but I don't see it anywhere, so maybe it didn't get saved? (I was on train wifi.) 
```suggestion
        echo -n "$indexImageResolved" | tee "$(results.indexImageResolved.path)"
```

I don't think you need `-e` - it enables special characters like `\n`, but you don't use those.

And `tee -a` will append to the file, but this is the first time we write to it, so it won't make a difference.

---

Indeed. The tee -a was for a previous trial to loop the images, which did not work. Thanks for the last last minute comments.

---

fixed.